lib/routes/register/index.js

```diff
@@ -159,13 +159,16 @@ async function register(server, options, done) {
     url: '/refresh',
     schema: schema.refresh,
     handler: async request => {
+      const { authorization } = request.headers
+      const refreshToken = authorization
+        ? authorization.replace(/^Bearer /, '')
```

Comment:
Do we have an issue with the trailing space, this w2ill be removed if no refresh token and then regex will not match?

Reply:
Yep, good catch